Common/VR/PPSSPPVR.cpp

```diff
@@ -8,7 +8,12 @@
 #include "Core/HLE/sceDisplay.h"
 #include "Core/Config.h"
 #include "Core/KeyMap.h"
+#include "Core/System.h"
 
+#include "GPU/OpenGL/GLRenderManager.h"
+#include "GPU/GPUState.h"
```

Comment:
Hm, a bit of a circular thing with Common depending on GPU...

-[Unknown]

Reply:
Yeah I think some of the VR stuff maybe should live in GPU/VR or something instead of in Common...